tw-tasks-core/src/main/java/com/transferwise/tasks/helpers/CoreMetricsTemplate.java

```diff
@@ -534,4 +536,14 @@ protected static class MetricHandle {
     private Meter meter;
 
   }
+
+  public void registerTaskCancelled(String taskType) {
+    meterCache.counter(METRIC_TASKS_CANCELLED_COUNT, TagsSet.of(TAG_BUCKET_ID, resolveBucketId(null), TAG_TASK_TYPE, taskType))
```

Comment:
There is no need to add the bucket tag at all.
Different (by name) metrics can have different set of tags.
And in dashboards the graphs will be "hand made" anyway.